src/app/page.tsx

```diff
@@ -7,15 +7,15 @@ import Footer from './components/Footer/Footer'
 import Image from 'next/image'
```

Comment:
check the import order:-

'use client'

import { useEffect, useState } from 'react'

import { BsStars } from 'react-icons/bs'
import Footer from './components/Footer/Footer'
import Image from 'next/image'
import Link from 'next/link'
import Navbar from './components/Navbar/Navbar'
import SearchBar from '../components/SearchBar'
import bacpacTitle from '../assets/bacpacTitle.png'
import bookImgLogo from '../assets/bookimg.png'
import client from '../client'
import discord from '../assets/discordLog.png'
import { query } from '../queries/queries'